ddtrace/ext/test.py

```diff
@@ -60,9 +57,16 @@
 # Test Codeowners
 CODEOWNERS = TEST_CODEOWNERS = "test.codeowners"
 
-ITR_TEST_SKIPPING_TYPE = "test.itr.tests_skipping.type"
+# ITR
+ITR_SKIPPED = "test.skipped_by_itr"
 
+# Test session-level ITR and coverage:
+ITR_DD_CI_ITR_TESTS_SKIPPED = "_dd.ci.itr.tests_skipped"
+ITR_TEST_SKIPPING_ENABLED = "test.itr.tests_skipping.enabled"
+ITR_TEST_SKIPPING_TESTS_SKIPPED = "test.itr.tests_skipping.tests_skipped"
+ITR_TEST_SKIPPING_TYPE = "test.itr.tests_skipping.type"
 ITR_TEST_SKIPPING_COUNT = "test.itr.tests_skipping.count"
+ITR_TEST_CODE_COVERAGE_ENABLED = "test.code_coverage.enabled"
```

Comment:
We also set these for modules, by the way

Reply:
Added in c47faba3f025d5c8b3cee05b1b684341fb88b3e9 .